### DISCUSSION TITLE
Define Embeds by Tag/Frontmatter/Folder

### DISCUSSION THREAD

**Discussion by ararrocks** [mProjectsCode/obsidian-meta-bind-plugin#379] [Ideas]
Hi, 

I always have the issue that when I change some structural stuff in my vault i need to edit most of my "database"-like files. To give an example: 

I have people notes in my vault. these develooped over time. some have dataview tables in them, some don't. some have dataview tables about meetings, others about activities, others logs - these differences came from changes in structure. 

having to regex replace in VS Code to bring all to par is cumbersome. especially when still trying to be flexible enough to work things out later and change stuff. 

Now i thought a plugin, where i can "embed" a template into these files and being able to change the template later would be great. the meta bind embed works for this VERY good, as references in the template relate to the file its embedded to. awesome!

but to make it really flexible, it would be great to have these embeds attached to a tag or other frontmatter key, rendered at a variable position (top, bottom, section) and maybe even multiple. think of it like the note-toolbar combined with meta bind embed. 

i tried to make a plugin but I cant get the embedding right. would you be able to point me into the right direction?

**Comment by ru-van-urk** [comment]
Did you find a solution?
